binder/binderlocationbackend.cpp

```diff
@@ -256,7 +258,7 @@ static GBinderLocalReply *geoclue_binder_gnss_callback(
                 loc.setAltitude(location->altitudeMeters);
 
             if (location->gnssLocationFlags & HYBRIS_GNSS_LOCATION_HAS_SPEED)
-                loc.setSpeed(location->speedMetersPerSec);
+                loc.setSpeed(location->speedMetersPerSec / KnotsToMps);
```

Comment:
Wouldn't it be more logical to have a mpsToKnots and multiply with it?

Reply:
I copied that from the old hal backend which for some reason had it like that. Maybe I should change both to mpsToKnots?